qiskit/ignis/experiments/calibration/types.py

```diff
@@ -31,6 +30,17 @@
 CalProg.circuits.__doc__ = 'List of calibration circuit.'
 CalProg.metadata.__doc__ = 'List of metadata representing experimental details.'
 
+CalValue = NamedTuple(
+    'CalValue',
+    [('value', Union[int, float, complex]),
+     ('validation', str),
+     ('timestamp', np.datetime64)]
+)
+CalValue.__doc__ = 'Calibrated value with timestamp and status of validation.'
+CalValue.value.__doc__ = 'Value'
+CalValue.validation.__doc__ = 'Calibration data validation status.'
+CalValue.timestamp.__doc__ = 'Time information of when this value is created.'
```

Comment:
Perhaps this should be in cal_table.py as that is very close to where it is used?

Reply:
This is to prevent cyclic import. Because type def can be imported by several modules, isolating defs can prevent conflict of import.